src/store/modules/segmentationModule/configuration.js

```diff
@@ -3,7 +3,7 @@ const configuration = {
   renderOutline: true,
   renderFill: true,
   shouldRenderInactiveLabelmaps: true,
-  radius: 10,
+  radius: 100,
```

Comment:
It's interesting. Switching from a CT to a DX, the brush size difference (to the end user) is very odd.
I wonder if other viewers maintain brush size in relation to the image's size.

Reply:
In slicer you can switch between mm and % of screen space.  Default is 3% of the size of the view in screen space and this seems to be a good default.  In practice, keeping the brush size fixed in screen space means that the size of the brush is effectively controlled by zooming in and out, which makes some sense because that ties it to the size of the feature you are looking at.

We used to also have an option for units of image voxels.  Sometimes it's important to turn individual voxels on. an off.

Reply:
Yeah I agree, its currently based in units of voxel size. This is something which is easy for us to change and make configurable. Zooming changing brush size does make a lot of sense.